ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -109,6 +109,8 @@ public class TransactionPool implements BlockAddedObserver {
   private final EthScheduler.OrderedProcessor<BlockAddedEvent> blockAddedEventOrderedProcessor;
   private final Map<VersionedHash, BlobsWithCommitments.BlobQuad> mapOfBlobsInTransactionPool =
       new HashMap<>();
+  private final ConcurrentHashMap<VersionedHash, BlobsWithCommitments.BlobQuad> blobCacheTracker =
+      new ConcurrentHashMap<>();
```

Comment:
what is this for? it seems to have no real usage

Reply:
It's being used to maintain the count of blobs in the cache...

Reply:
better to add a `size` method to `BlobCache`, it is simpler and avoid keeping this extra map

Reply:
Okay, I was trying to avoid doing that... I have added it now and removed the extra map